app/models/sac_cas/person.rb

```diff
@@ -107,9 +100,23 @@ def sac_membership
 
   private
 
+  def set_digital_correspondence
+    self.correspondence = "digital"
+  end
+
+  def password_initialized?
+    encrypted_password_changed? && encrypted_password.present? && encrypted_password_was.blank?
+  end
+
   def check_data_quality
-    return if (People::DataQualityChecker::ATTRIBUTES_TO_CHECK & saved_changes.keys).empty?
+    People::DataQualityChecker.new(self).check_data_quality if
+      (People::DataQualityChecker::ATTRIBUTES_TO_CHECK & saved_changes.keys).any?
+  end
```

Comment:
Ist bikeshedding, aber ein `if` mit der Condition auf der nächsten Zeile schreiben wir sonst kaum so. Ich finde die Variante mit dem early return, wie sie vorher war, deutlich klarer. Insbesondere, wenn es dann wie in `transmit_data_to_abacus` noch mehrzeilig wird.